core/src/trezor/ui/layouts/tt/__init__.py

```diff
@@ -983,33 +982,26 @@ async def confirm_sign_identity(
 
 
 async def confirm_signverify(
-    ctx: wire.GenericContext, coin: str, message: str, address: str | None = None
+    ctx: wire.GenericContext, coin: str, message: str, address: str, verify: bool
 ) -> None:
-    if address:
+    if verify:
         header = f"Verify {coin} message"
-        font = ui.MONO
         br_type = "verify_message"
-
-        text = Text(header, new_lines=False)
-        text.bold("Confirm address:\n")
-        text.mono(*chunks_intersperse(address, MONO_ADDR_PER_LINE))
-        await raise_if_cancelled(
-            interact(ctx, Confirm(text), br_type, ButtonRequestType.Other)
-        )
     else:
         header = f"Sign {coin} message"
-        font = ui.NORMAL
         br_type = "sign_message"
 
+    text = Text(header, new_lines=False)
```

Comment:
this should be using `confirm_address()` layout

Reply:
...i wanted to make an issue to fix consistency of this, but it's probably not worth it while we expect to move to Rust UI.